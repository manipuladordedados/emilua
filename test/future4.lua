local future = require 'future'

local p, f = future.new()
print(f:get())
print('never gets here')
