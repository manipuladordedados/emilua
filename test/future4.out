Possible deadlock(s) detected during VM 0x1 shutdown:
	Underlying promise for future 0x2 is broken
